dark/filter.py

```diff
@@ -116,6 +116,93 @@ def accept(self, title):
         return self.DEFAULT_ACCEPT
 
 
+class SequenceFilter:
+    """
+    Provide an acceptance test for sequences.
+
+    @param whitelist: If not C{None}, a C{set} of exact sequences that are always
+        acceptable.
+    @param blacklist: If not C{None}, a C{set} of exact sequences that are never
+        acceptable.
+    @param whitelistFile: If not C{None}, a C{str} filename containing lines
+        that give exact sequences that are always acceptable.
+    @param blacklistFile: If not C{None}, a C{str} filename containing lines
+        that give exact sequences that are never acceptable.
+    @param positiveRegex: If not C{None}, a C{str} regex that sequence sequences
+        must match (case is ignored).
+    @param negativeRegex: If not C{None}, a C{str} regex that sequence sequences
+        must not match (case is ignored).
+    """
+
+    REJECT = 0
```

Comment:
Don't you need to put these on `self`?

Reply:
They are class attributes, and are accessed via `self`. See the code later in the class.